### DISCUSSION TITLE
Outdated Drizzle documentation

### DISCUSSION THREAD

**Discussion by NathanBrodin** [vercel/community#7102] [Bug Report]
### Page to Investigate

https://vercel.com/docs/storage/vercel-postgres/using-an-orm#drizzle

### Steps to Reproduce

Hello, as stated in the official Drizzle documentation [How to migrate to `0.21.0`](https://orm.drizzle.team/kit-docs/upgrade-21#how-to-migrate-to-0210), the `drizzle.config.ts` needs the following changes:

```ts
import "@/drizzle/envConfig";
import { defineConfig } from "drizzle-kit";

export default defineConfig({
  schema: "./drizzle/schema.ts",
  dialect: "postgresql",
  dbCredentials: {
    url: process.env.POSTGRES_URL!,
  },
});
```

We need to replace `connectionString` by `url` and replace `driver: 'pg'` by dialect: "postgresql".

Also, the command `npx drizzle-kit push:pg` is now deprecated and needs to be replaced by `npx drizzle-kit push`

Hope this helps! The Vercel documentation is not on GitHub, so I couldn't make the change myself, but I have provided the updated docs here!

**Comment by amyegan** [comment]
Thanks for reporting this! I've shared it with our documentation team so we can get that example updated.

[a comment by an automated account is not shown]
